src/features/vote/CommitPhase.tsx

```diff
@@ -375,15 +368,11 @@ const CommitPhase: FC<Props> = ({
         isOpen={isOpen}
         close={close}
         ref={modalRef}
-        modalState={modalState}
-        setModalState={setModalState}
```

Comment:
how come this was no longer needed? just curious

Reply:
the modal state doesn't have the pending and success states, which were related to the transaction, which is now handled by Notify.